packages/devtools_app/pubspec.yaml

```diff
@@ -123,12 +121,12 @@ flutter:
         - asset: packages/codicon/font/codicon.ttf
 
 dependency_overrides:
-# The "#OVERRIDE_FOR_DEVELOPMENT" lines are stripped out when we publish.
+# The '#OVERRIDE_FOR_DEVELOPMENT' lines are stripped out when we publish.
 # All overriden dependencies are published together so there is no harm
 # in treating them like they are part of a mono-repo while developing.
   devtools_server: #OVERRIDE_FOR_DEVELOPMENT
-    path: ../devtools_server  #OVERRIDE_FOR_DEVELOPMENT
-  devtools_shared:  #OVERRIDE_FOR_DEVELOPMENT
-    path: ../devtools_shared  #OVERRIDE_FOR_DEVELOPMENT
-  devtools_testing:  #OVERRIDE_FOR_DEVELOPMENT
-    path: ../devtools_testing  #OVERRIDE_FOR_DEVELOPMENT
+    path: ../devtools_server #OVERRIDE_FOR_DEVELOPMENT
```

Comment:
We'll need to verify that the 
too/publish.sh script still strips these lines out. Otherwise this cleanup is fine.